app/com/m3/octoparts/repository/config/CacheGroupRepository.scala

```diff
@@ -33,12 +35,12 @@ object CacheGroupRepository extends ConfigMapper[CacheGroup] with TimestampsFeat
     many = HttpPartConfigRepository,
     throughFk = "cacheGroupId",
     manyFk = "httpPartConfigId",
-    merge = (group, partConfigs) => group.copy(httpPartConfigs = partConfigs)
+    merge = (group, partConfigs) => group.copy(httpPartConfigs = partConfigs.sorted)
```

Comment:
1. I think it is most efficient to sort it as soon as possible, since the results of the findXXX may be cached. It might also be convenient to have the output of the list endpoints consistent.

2. I wish I knew a simple way to do it with the ORM :)